Assets/Prefabs/Meteor/Meteor01.prefab

```diff
@@ -0,0 +1,81 @@
+%YAML 1.1
```

Comment:
Could you move the meteor prefabs into a VFX subfolder? along with all the other VFX prefabs